packages/typespec-python/package.json

```diff
@@ -43,34 +43,30 @@
     "get-autorest-python-path.cjs"
   ],
   "peerDependencies": {
-    "@typespec/versioning": "~0.43.0",
-    "@typespec/rest": "~0.43.0",
-    "@typespec/http": "~0.43.1",
-    "@azure-tools/typespec-azure-core": "~0.29.0"
+    "@typespec/compiler": ">=0.43.0 <1.0.0",
+    "@typespec/versioning": ">=0.43.0 <1.0.0",
+    "@typespec/rest": ">=0.43.0 <1.0.0",
+    "@typespec/http": ">=0.43.1 <1.0.0",
+    "@azure-tools/typespec-azure-core": ">=0.29.0 <1.0.0"
   },
   "dependencies": {
     "js-yaml": "~4.1.0",
     "@autorest/python": "workspace:^",
-    "@typespec/compiler": "~0.43.0",
     "@azure-tools/typespec-client-generator-core": "~0.30.0-dev.10"
```

Comment:
that one also should be a `peerDependencies`. Might also want it in `devDependencies` if you want to pick which version to use local (specially for `.dev`)

Reply:
Sounds good